src/py42/services/detectionlists/high_risk_employee.py

```diff
@@ -41,6 +41,7 @@ def add(self, user_id):
 
         Raises a :class:`Py42BadRequestError` when a user already exists in the High Risk Employee
         detection list.
+        `REST Documentation <https://developer.code42.com/api/#operation/HighRiskEmployeeControllerV2_AddEmployee>`__
 
         Args:
             user_id (str or int): The Code42 userUid of the user you want to add to the High Risk
```

Comment:
Could we add the method links for High Risk Employee as well?

Reply:
added to almost all methods.